app/scripts/formentry/modules/rest-services/location.service.js

```diff
@@ -112,5 +113,63 @@ jshint -W003,-W109, -W106, -W098, -W003, -W068, -W004, -W033, -W030, -W117, -W11
           console.error(error);
         });
     }
-  }
+    
+    //hardcoded locations for testing
```

Comment:
You can get a better place to put this test locations and still make your method o use them